app/javascript/components/LogInPage.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "antd/dist/antd.css";
+import Utils from 'helpers/utils';
+import { Form, Icon, Input, Button, Checkbox } from 'antd';
+import '../../assets/stylesheets/loginpage.css';
+var logo = require('../../assets/images/logo.png');
+
+class LogInPage extends React.Component {
+ constructor(props) {
+   super(props);
+   this.state = {
+     email: '',
+     password: '',
+     remember_me: 0,
+     errorMessage: '',
+     hasError: false
+   }
+ }
+
+ handleChange = (event) => {
+   const name = event.target.name;
+   const value = event.target.value;
+   if (name === "email") {
+     this.setState({ email: value });
+   } else {
+     this.setState({ password: value });
+   }
+ }
+
+ handleRememberMe = (event) => {
+   this.setState({ remember_me: !this.state.remember_me });
```

Comment:
do "1 - this.state.remember_me" here to keep the value an integer.

Reply:
done!